http/headers/Content-Disposition.json

```diff
@@ -13,7 +13,8 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "When saving documents, the document title is used instead of the `filename` parameter if the disposition type is `inline` [bug 352093465](https://crbug.com/352093465)."
```

Comment:
Can you add the same note to Safari, which seems to have the same behavior?

Reply:
I found https://bugs.webkit.org/show_bug.cgi?id=18384 which appears to be an equivalent report

Reply:
Added in 3df0c7e8576b7ff2cacfeeae32721d76e308ead3